src/components/ArticleList.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { useState } from "react";
+import { useEffect } from "react";
+import axios from "axios";
+import ArticleCard from "./ArticleCard";
+import { Link } from "react-router-dom";
+
+function ArticleList() {
+  const [articles, setArticles] = useState([]);
+  useEffect(() => {
+    axios
+      .get("https://milos-nc-news.onrender.com/api/articles")
```

Comment:
this request should be in a separate `api-utils` file to keep your components clean